### PR TITLE
Entry points can also be defined in `setup.cfg`

### PR DESCRIPTION
**Pull request by nsoranzo**
E.g. https://github.com/pypa/virtualenv/blob/master/setup.cfg#L61

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
/rant 
I really miss the "there is only one obvious way" philosophy in Python. The packaging world is becoming the real life version of the XKCD standards cartoon.
rant/

File: src/maintainer/knowledge_base.rst [hunk 1]
```diff
@@ -687,11 +687,11 @@ In order to qualify as a noarch python package, all of the following criteria mu
   - No skips except for python version. If the recipe is py3 only, remove skip
     statement and add version constraint on python in ``host`` and ``run``
     section.
-  - 2to3 is not used
+  - ``2to3`` is not used
   - Scripts argument in setup.py is not used
-  - If ``console_script`` entrypoints are in setup.py, they are listed in meta.yaml
+  - If ``console_script`` entry points are defined in ``setup.py`` or ``setup.cfg``, they are also listed in the ``build`` section of ``meta.yaml``
```
Comment (ocefpaf) [inline src/maintainer/knowledge_base.rst]: I would keep it one word b/c that is how people will find it in the `setup.*` files.

Comment (nsoranzo) [reply src/maintainer/knowledge_base.rst]: The world is actually `entry_points`, which I had used at a certain point while editing this, but then I have to also quote it as code right after another code-quoted word. But I can do that if you prefer.

Comment (ocefpaf) [reply src/maintainer/knowledge_base.rst]: Indeed. I think it is better.

**Pull request merged**